src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -48,10 +49,20 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <ItemGroup>
+    <TestTargetFrameworkPath Include="$(TestPath)%(TestTargetFramework.Folder)"/>
```

Comment:
While right now we only have one TestTrargeFramework the goal is to have more in he future and by moving this outside the target you have broken the batching that as occurring in the target. You may need to actually do the file path building inline in the target (even if it is duplicated).